app/controllers/login_controller.rb

```diff
@@ -24,9 +24,9 @@ def index
 
   def login
     if params[:email].to_s.match(/@/)
-      user = User.where(:email => params[:email]).first
+      user = User.where('lower(email) = ?', params[:email].downcase).first
     else
-      user = User.where(:username => params[:email]).first
+      user = User.where('lower(email) = ?', params[:email].downcase).first
```

Comment:
You probably meant to write `lower(username) = ?` in this SQL statement, which is why Rubocop is going after you to move the identical statements out of the conditional.

Also, oddly enough while I was testing out the case insensitivity, I found that I can already log in with a case insensitive username, likely due to MySQL column type `utf8mb4_general_ci` (https://github.com/lobsters/lobsters/blob/master/db/schema.rb#L245)